app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
@cben So you are suggesting accepting any input and copying them straight to the queue?

I view the purpose of methods like these to dictate and protect the messages that go onto the queue.

If you are going to document what parameters are going into `*cb_method` then why not just document it by having actual variables?

But I was having trouble understanding the full implications of comment from @cben - sorry if I totally botched it.